api/bibliotheca.py

```diff
@@ -247,7 +247,8 @@ def request(self, path, body=None, method="GET", identifier=None,
         else:
             return self._request_with_timeout(
                 method, url, data=body, headers=headers,
-                allow_redirects=False)
+                allow_redirects=False, timeout=60
+            )
```

Comment:
The Bibliotheca API was giving me a lot of timeouts when I tested this, so I boosted the default timeout from 20 minutes to 60. Hopefully this will resolve other transient Bibliotheca API failures going forward.